dlp/snippets/src/main/java/dlp/snippets/InspectGcsFileSendToScc.java

```diff
@@ -0,0 +1,116 @@
+    // The name of the file in the bucket
+    String gcsPath = "gs://" + "your-bucket-name" + "path/to/file.txt";
+    createJobSendToScc(projectId, gcsPath);
+  }
+
+  // Creates a DLP Job
+  public static void createJobSendToScc(String projectId, String gcsPath) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      // Specify the GCS file to be inspected.
+      CloudStorageOptions cloudStorageOptions =
+          CloudStorageOptions.newBuilder()
+              .setFileSet(CloudStorageOptions.FileSet.newBuilder().setUrl(gcsPath))
+              .build();
+
+      StorageConfig storageConfig =
+          StorageConfig.newBuilder()
+              .setCloudStorageOptions(cloudStorageOptions)
+              .build();
+
+      // Specify the type of info the inspection will look for.
+      // See https://cloud.google.com/dlp/docs/infotypes-reference for complete list of info types
+      List<InfoType> infoTypes =
+          Stream.of("EMAIL_ADDRESS", "PERSON_NAME", "LOCATION", "PHONE_NUMBER")
+              .map(it -> InfoType.newBuilder().setName(it).build())
+              .collect(Collectors.toList());
+
+      // The minimum likelihood required before returning a match:
+      // See: https://cloud.google.com/dlp/docs/likelihood
+      Likelihood minLikelihood = Likelihood.UNLIKELY;
+
+      // The maximum number of findings to report (0 = server maximum)
+      InspectConfig.FindingLimits findingLimits =
+          InspectConfig.FindingLimits.newBuilder().setMaxFindingsPerItem(100).build();
+
+      InspectConfig inspectConfig =
+          InspectConfig.newBuilder()
+              .addAllInfoTypes(infoTypes)
+              .setIncludeQuote(true)
+              .setMinLikelihood(minLikelihood)
+              .setLimits(findingLimits)
+              .build();
+
+      // Specify the action that is triggered when the job completes.
+      Action.PublishSummaryToCscc publishSummaryToCscc =
+          Action.PublishSummaryToCscc.getDefaultInstance();
+      Action action = Action.newBuilder().setPublishSummaryToCscc(publishSummaryToCscc).build();
+
+      // Configure the inspection job we want the service to perform.
+      InspectJobConfig inspectJobConfig =
+          InspectJobConfig.newBuilder()
+              .setInspectConfig(inspectConfig)
+              .setStorageConfig(storageConfig)
+              .addActions(action)
+              .build();
+
+      // Construct the job creation request to be sent by the client.
+      CreateDlpJobRequest createDlpJobRequest =
+          CreateDlpJobRequest.newBuilder()
+              .setParent(LocationName.of(projectId, "global").toString())
```

Comment:
nit: Consider moving "global" to a variable in main method.

Reply:
Generally, we are not declaring "global" in main() for all the code samples.
So if I make changes in current PR, we need to incorporate the changes for all the code sample thus I'm not making the requested changes.